src/argilla/cli/server/database/users/create.py

```diff
@@ -27,6 +26,7 @@
     UserCreate,
     WorkspaceCreate,
 )
+from argilla.utils.pydantic import constr
```

Comment:
Since this `utils.pydantic` is used just to contro how pydantic is imported, I would propose a different naming. `argilla.imports.pydantic` or just `argilla._import_pydantic`